lib/puppet/provider/certs_bootstrap_rpm/katello_ssl_tool.rb

```diff
@@ -33,7 +33,17 @@ def run
   protected
 
   def last_rpm
-    Dir.glob(File.join(resource[:dir], "#{resource[:name]}-*.noarch.rpm")).sort.last
+    rpms = Dir.glob(File.join(resource[:dir], "#{resource[:name]}-*.noarch.rpm"))
+
+    rpms = rpms.collect do |rpm|
+      rpm_split = rpm.split("#{resource[:name]}-")[1].split('.noarch.rpm')[0]
+      version = rpm_split.split('-')[0]
+      release = rpm_split.split('-')[1]
+
+      {'release' => release, 'rpm' => rpm}
+    end
+
+    rpms.sort { |a,b| a['release'].to_i <=> b['release'].to_i }.last
```

Comment:
Does the RPM version never change, always 1.0? Could that change some day?

The last RPM ends up as 1.0.11 when we have: 

```ruby
["foo-1.0-1.noarch.rpm", "foo-2.0-2.noarch.rpm", "foo-1.0-11.noarch.rpm"]
```